products/azure-kubernetes-service.md

```diff
@@ -12,36 +12,40 @@ releaseColumn: false
 releaseDateColumn: true
 eolColumn: End of Support
 
+# releaseDate can be found by following each cycle's link.
+# Not sure corresponding cycles will be added one day, but here are the announcements for:
+# 1.20: https://azure.microsoft.com/en-us/updates/general-availability-kubernetes-v120-support-in-aks/
+# 1.19: https://azure.microsoft.com/en-us/updates/aks-updates-for-november-2020/
```

Comment:
Better put these 3 lines below `releaseCycle: "1.21"` or add the cycles.

Reply:
I put them there because I did not know if it would work with auto-update. But there is no auto-update yet for that product, so moved this comment below 1.21 as requested.